packages/visx-demo/src/sandboxes/visx-tooltip/Example.tsx

```diff
@@ -205,7 +237,49 @@ export default function Example({ width, height, showControls = true }: TooltipP
           padding: 16px;
           line-height: 1.2em;
         }
+        .overlay-root {
+          z-index: 3000;
+          position: relative;
+        }
+        .overlay-layer {
+          position: absolute;
+          border-radius: 8px;
+          padding: 8px;
+        }
+        .overlay-under-tooltip {
+          top: 30px;
+          right: 10px;
+          background: rgba(52, 235, 180, 0.8);
+        }
+        .overlay-over-tooltip {
+          top: 70px;
+          right: 30px;
+          background: rgba(250, 235, 180, 0.8);
+        }
       `}</style>
     </>
   );
 }
+
+type OverlayLayerProps = {
+  container: HTMLDivElement | null;
+  text: string;
+  className?: string;
+  placeAfterTooltipInDom?: boolean;
+};
+
+function OverlayLayer({ className, container, placeAfterTooltipInDom, text }: OverlayLayerProps) {
```

Comment:
maybe I'm not fully following the nuance, but I find that these make the example much more complex to understand / they don't directly relate to the tooltip API that we want to demonstrate. wdyt about removing them?

Reply:
I see your point about this making the example code more complex, but on the other hand I feel like having these elements in the example helps understand what the `portalContainer` prop in `useTooltipInPortal` can achieve and I'm worried that the point may be missed if we removed this from the example. 🤔 Plus, I think it's a nice way to visually make sure that nothing is broken when making changes to the tooltip module. I won't fight too hard if you feel strongly about trimming the example though. 🤷‍♂️